### DISCUSSION TITLE
How to using PDF file as input type?

### DISCUSSION THREAD

**Discussion by JonWangM** [microsoft/graphrag#950] [Q&A]
Will the team have a PDF as input to the development plan?

**Comment by akashAD98** [comment]
as of now its supporting text/csv . you can convert your pdf into text file & it ill work

**Reply by JonWangM** [reply]
As we know, PDF contains a variety of data information, such as tables, pictures, citations, chapters, etc., if we convert to txt, will we lose a lot of information?

**Comment by l4b4r4b4b4** [comment]
I use unstructured to ingest pdfs and construct txt files in `input`.

**Reply by AlexCastroAlex** [reply]
Hello , Did you use API deployed or front end ? 
And where did you edit code to use unstructured  ? 
That's a real lack that a graph rag can not ingest many types of files like AI Search can 😢

**Reply by l4b4r4b4b4** [reply]
i used unstructured library if thats what you mean with the first question.

I did not change any graphrag code. Simply used unstructured to create txt from pdfs.

**Reply by AlexCastroAlex** [reply]
Well ok i see. 
Thanks for answer 😊

**Comment by yaroslavyaroslav** [comment]
Batch images (of pdf) processing with gpt-4o-mini costs me around 1$ for everything (a single yet complex tech book of 300 pages)  and provide quite impressive result.
